_data/contributors.yml

```diff
@@ -1,3 +1,13 @@
+- name: Ariane Sasso
+  bio: 'Researcher in Medical Informatics and Digital Health'
+  organization: "Hasso Plattner Institute"
+  github_username: arianesasso
+  github_image_id: 3659681 # You can find this by right clicking on the image in your bio, and copying the link. the last part contains a 7 digit number that is your avatar image!
```

Comment:
probably we don't need to keep the comment here about the github_image_id

Other than that, LGTM! Thank you so much @arianesasso !